tools/check_release_notes.sh

```
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

nit: it may be beneficial to enable **errexit** with`set -e`; it could help remove the need to do some of the `$?` checks and/or catch other miscellaneous errors.

---

Added `set -e`. Had to disable it for grep or it would fail silently. Thanks.